main.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  conditions = merge(var.additional_conditions, {
+    aud        = var.openid_audience
```

Comment:
Isn't this always `"token.actions.githubusercontent.com:aud": "sts.amazonaws.com"`? Why would we want to make the audience configuration?

Reply:
@awendt from https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#understanding-the-oidc-token

> `aud`: (Audience) By default, this is the URL of the repository owner, such as the organization that owns the repository. This is the only claim that can be customized. You can set a custom audience with a toolkit command: [core.getIDToken(audience)](https://www.npmjs.com/package/@actions/core/v/1.6.0)

Thus, this is hard-coded in `aws-actions/configure-aws-credentials`

https://github.com/aws-actions/configure-aws-credentials/blob/master/index.js#L313

but it is technically possible to pass a different value from the GitHub Actions job to `sts:AssumeRoleWithWebIdentity`, if you'd not use `aws-actions/configure-aws-credentials`.

Reply:
@jansiwy Got it, thanks!